src/ImageSharp/Formats/Tiff/TiffDecoderMetadataHelpers.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Collections.Generic;
+using System.Linq;
+
+using SixLabors.ImageSharp.Formats.Experimental.Tiff.Constants;
+using SixLabors.ImageSharp.Metadata;
+using SixLabors.ImageSharp.Metadata.Profiles.Exif;
+using SixLabors.ImageSharp.Metadata.Profiles.Icc;
+using SixLabors.ImageSharp.Metadata.Profiles.Iptc;
+
+namespace SixLabors.ImageSharp.Formats.Experimental.Tiff
+{
+    /// <summary>
+    /// The decoder metadata helper methods.
+    /// </summary>
+    internal static class TiffDecoderMetadataHelpers
+    {
+        public static ImageMetadata CreateMetadata(this IList<TiffFrameMetadata> frames, bool ignoreMetadata, ByteOrder byteOrder)
+        {
+            var coreMetadata = new ImageMetadata();
+
+            TiffFrameMetadata rootFrameMetadata = frames.First();
+            switch (rootFrameMetadata.ResolutionUnit)
```

Comment:
`TiffFrameMetadata` should use `PixelResolutionUnit`

Reply:
Done!